bridges/snowbridge/pallets/inbound-queue-v2/src/lib.rs

```diff
@@ -208,6 +209,10 @@ pub mod pallet {
 	#[pallet::storage]
 	pub type OperatingMode<T: Config> = StorageValue<_, BasicOperatingMode, ValueQuery>;
 
+	/// Keep track of tips added for a message as an additional relayer incentivization.
+	#[pallet::storage]
+	pub type Tips<T: Config> = StorageMap<_, Twox64Concat, u64, u128, ValueQuery>;
```

Comment:
I think this storage item is easily manipulated by users, so we need to use a stronger hasher than `Twox64Concat`.

Reply:
https://github.com/vgeddes/polkadot-sdk/pull/23/commits/d1a5ab47740706370c520b8a237c37580a32a05e